service/storage_service.hh

```diff
@@ -370,6 +378,7 @@ public:
     // should only be called via JMX
     future<> stop_gossiping();
 
+    // Allows recovery from a forcibly killed node.
```

Comment:
What does this mean?

Reply:
No idea. I copied it from api/api-doc/storage_service.json:
```
            {
               "method":"POST",
               "summary":"allows a user to recover a forcibly 'killed' node",
               "type":"void",
               "nickname":"start_gossiping",
               "produces":[
                  "application/json"
               ],
               "parameters":[
               ]
            },
```

does anyone ever use this? What is this for?

But yeah, this patchset is probably not the place for adding such comments

Reply:
Removed the comment